lib/response.js

```diff
@@ -113,10 +114,20 @@ res.send = function send(body) {
   var encoding;
   var req = this.req;
   var type;
+  var isChunkBlob = chunk && chunk.constructor && chunk.constructor.name === 'Blob';
```

Comment:
is there a reason not to use instanceof operator, instead of constructor.name ?
it would allow the user to send a custom blob extending Blob if need be

Reply:
yes you are right just it